docs/core/resilience/http-resilience.md

```diff
@@ -81,13 +81,13 @@ The preceding code adds the standard resilience handler to the <xref:System.Net.
 
 The default configuration chains five resilience strategies in the following order (from the outermost to the innermost):
 
-| Order | Strategy | Description |
-|--:|--|--|
-| **1** | Rate limiter | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. |
-| **2** | Total request timeout | The total request timeout pipeline applies an overall timeout to the execution, ensuring that the request, including retry attempts, doesn't exceed the configured limit. |
-| **3** | Retry | The retry pipeline retries the request in case the dependency is slow or returns a transient error. |
-| **4** | Circuit breaker | The circuit breaker blocks the execution if too many direct failures or timeouts are detected. |
-| **5** | Attempt timeout | The attempt timeout pipeline limits each request attempt duration and throws if it's exceeded. |
+| Order | Strategy | Description | Defaults |
+|--:|--|--|--|
+| **1** | Rate limiter | The rate limiter pipeline limits the maximum number of concurrent requests being sent to the dependency. | Queue: 1,000<br>Permit: 0 |
```

Comment:
Isn't this the opposite way around? [Code](https://github.com/App-vNext/Polly/blob/9ca23fe8f38f3885566a5105e3779024ddbd049e/src/Polly.RateLimiting/RateLimiterConstants.cs#L7-L9)